tests/sanity/tests/actions.spec.ts

```diff
@@ -10,19 +10,39 @@ test.describe('actions tests', () => {
     // Create user and workspace
     await (await page.goto(`${PlatformURI}/workbench/sanity-ws/sanity-ws`))?.finished()
   })
-  test('action-new-candidate', async ({ page }) => {
-    await page.click('[id="app-recruit\\:string\\:RecruitApplication"]')
 
-    await page.click('text=Talents')
-    await expect(page).toHaveURL(`${PlatformURI}/workbench/sanity-ws/recruit/talents`)
+  test('Add new candidate', async ({ page, authorizedMainPage, recruitPage }) => {
+    await test.step(`Open recruit application`, async () => {
+      await authorizedMainPage.leftSidebar.openRecruitApplication();
+    });
 
-    await page.click('td:has-text("Frontend Engineer")')
+    await test.step(`Open talents tab`, async () => {
```

Comment:
Wrapping every statement in a separate step seem to be excessive to me. I would prefer larger steps.